scripts/update-reg-repo.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Write output to error output stream.
+echo_stderr() {
+  echo -e "${1}" >&2
+}
+
+die()
+{
+  local _return="${2:-1}"
+  echo_stderr "$1"
+  exit "${_return}"
+}
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/.."
+CHART_DIR="$ROOT_DIR/deploy/helm/charts"
+VALUES_YAML="$CHART_DIR/values.yaml"
+
+NEW_REGISTRY="ghcr.io"
+NEW_REPOSITORY="openebs/dev"
+
+source "$SCRIPT_DIR/yq_utils.sh"
+
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [OPTIONS]
+
+Options:
+  --registry                                The registry to be updated to.
+  --repository                              The repository to be updated to.
+
+Examples:
+  $(basename "$0") --registry ghcr.io --repository openebs/dev
```

Comment:
this here seems to be the namespace

Reply:
Yes it is the namespace but if you see the values.yaml it calls it repository here.

Reply:
no, on the values it is the repository it's `openebs/zfs-driver` which is namespace + image